app/locale/locale.coffee

```diff
@@ -76,7 +76,10 @@ Object.defineProperties module.exports,
   load:
     enumerable: false
     value: (langCode) ->
-      return Promise.resolve() if langCode in ['en', 'en-US']
+      if langCode in ['en', 'en-US']
+        @storeLoadedLanguage(langCode, module.exports[langCode])
```

Comment:
Does this work because the `en` locale is loaded by default? Also love this refactor because I wouldn't expect the `en` locale to not be available in the store.

Reply:
Yup en is loaded by default with an explicit require a little higher up in the file, so this change ensures that the loaded locales state is accurate.